modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/test/BatchEngineBundleTrackerTest.java

```diff
@@ -154,6 +161,81 @@ public void testProcessBatchEngineBundleOnUpgrade() throws Exception {
 		}
 	}
 
+	@Test
+	@TestInfo("LPD-61755")
+	public void testProcessBatchEngineBundleUsesActiveAdministratorUser()
+		throws Exception {
+
+		long originalUserId = -1;
+		int originalStatus = -1;
+		User fallbackAdminUser = null;
+
+		try {
+
+			// First batch engine run to capture initial user
+
+			AtomicReference<BatchEngineImportTask> initialTaskRef =
+				new AtomicReference<>();
+
+			_testProcessBatchEngineBundle(
+				batchEngineImportTask -> initialTaskRef.compareAndSet(
+					null, batchEngineImportTask),
+				"batch11", "/batch11/data.batch-engine-data.json");
+
+			BatchEngineImportTask initialTask = initialTaskRef.get();
+
+			originalUserId = initialTask.getUserId();
+
+			User originalUser = _userLocalService.getUser(originalUserId);
+
+			Assert.assertTrue(originalUser.isActive());
+
+			originalStatus = originalUser.getStatus();
+
+			// Simulate a scenario where the original user becomes inactive
+
+			_userLocalService.updateStatus(
+				originalUserId, WorkflowConstants.STATUS_INACTIVE,
+				new ServiceContext());
+
+			Assert.assertFalse(originalUser.isActive());
+
+			// Create an active admin user as a fallback for batch processing
+
+			fallbackAdminUser = _createAdminUser();
+
+			Assert.assertTrue(fallbackAdminUser.isActive());
+
+			// Second batch engine run to verify user selection
+
+			AtomicReference<BatchEngineImportTask> fallbackTaskRef =
+				new AtomicReference<>();
+
+			_testProcessBatchEngineBundle(
+				batchEngineImportTask -> fallbackTaskRef.compareAndSet(
+					null, batchEngineImportTask),
+				"batch11", "/batch11/data.batch-engine-data.json");
+
+			BatchEngineImportTask fallbackTask = fallbackTaskRef.get();
+
+			User fallbackUser = _userLocalService.getUser(
+				fallbackTask.getUserId());
+
+			Assert.assertTrue(
+				"Fallback user should be active", fallbackUser.isActive());
```

Comment:
I think we usually don't set comments in the assertTrue assertion, the line is already descriptive and BC always removes it